pipeline/mutate/mutator_id_token.go

```diff
@@ -41,23 +44,29 @@ type MutatorIDTokenRegistry interface {
 }
 
 type CredentialsIDTokenConfig struct {
-	Audience []string `json:"aud"`
+	Audience []string          `json:"aud"`
```

Comment:
I guess the question is if we want to keep this or not. As now we can use the claims portion instead.

Reply:
True. Do you want me to delete it?

Reply:
Yeah, I think that makes sense - but we have to document that in the upgrade guide .)

Reply:
Sure! I'm on it!